README.md

```diff
@@ -11,7 +11,7 @@
 
 👤 **Author1**
 
-- GitHub: [@til2to](https://github.com/til2to)
+- GitHub: [@til2to](https://til2to.github.io/porfolio_projects/)
```

Comment:
- [ ] Optional: Kindly make sure to place the deploy link under Live. You have mistakenly replaced the author GitHub link.